.github/workflows/python-test.yml

```diff
@@ -27,7 +27,17 @@ jobs:
     - name: Test with pytest
```

Comment:
why do we have --max-line-length=100 twice in the flake command?

Reply:
I can remove

Reply:
Removed the duplicated command

Reply:
Thanks!